app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  def taunt_police4
+    pp "taunt police"
```

Comment:
```suggestion
    pp 'taunt police'
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>